Description: missing renames of scst_itf_ver.h to iscsi_scst_itf_ver.h
 .
 scst (3.0.0~pre2+svn5854-ppa1) wheezy; urgency=high
 .
   * Include debian directory from r5165
     + Updated to svn5854
Author: Turbo Fredriksson <turbo@bayour.com>

---
Index: scst_svn/scst.spec.in
===================================================================
--- scst_svn.orig/scst.spec.in	2015-03-27 17:53:43.381425111 +0100
+++ scst_svn/scst.spec.in	2015-03-27 17:54:06.073139836 +0100
@@ -184,7 +184,7 @@
 /usr/include/scst/scst.h
 /usr/include/scst/scst_const.h
 /usr/include/scst/scst_debug.h
-/usr/include/scst/scst_itf_ver.h
+/usr/include/scst/iscsi_scst_itf_ver.h
 /usr/include/scst/scst_sgv.h
 /usr/include/scst/scst_user.h
 
Index: scst_svn/scst/src/Makefile
===================================================================
--- scst_svn.orig/scst/src/Makefile	2015-03-27 17:53:43.381425111 +0100
+++ scst_svn/scst/src/Makefile	2015-03-27 17:54:06.073139836 +0100
@@ -77,7 +77,7 @@
 SCST_INC_DIR := ../include
 endif
 
-SCST_INTF_VER_FILE := $(SCST_INC_DIR)/scst_itf_ver.h
+SCST_INTF_VER_FILE := $(SCST_INC_DIR)/iscsi_scst_itf_ver.h
 
 $(SCST_INTF_VER_FILE): $(SCST_INC_DIR)/scst.h $(SCST_INC_DIR)/scst_const.h $(SCST_INC_DIR)/scst_user.h
 	echo "/* Autogenerated, don't edit */" >$(SCST_INTF_VER_FILE)
@@ -116,7 +116,7 @@
 	install -m 644 ../include/scst_debug.h $(INSTALL_DIR_H)
 	install -m 644 ../include/scst_user.h $(INSTALL_DIR_H)
 	install -m 644 ../include/scst_const.h $(INSTALL_DIR_H)
-	install -m 644 ../include/scst_itf_ver.h $(INSTALL_DIR_H)
+	install -m 644 ../include/iscsi_scst_itf_ver.h $(INSTALL_DIR_H)
 	rm -f $(INSTALL_DIR_H)/$(MODULE_SYMVERS)
 	install -m 644 $(MODULE_SYMVERS) $(INSTALL_DIR_H)
 	-/sbin/depmod -b $(INSTALL_MOD_PATH)/ -a $(KVER)
Index: scst_svn/scst/include/scst_const.h
===================================================================
--- scst_svn.orig/scst/include/scst_const.h	2015-03-27 17:53:43.381425111 +0100
+++ scst_svn/scst/include/scst_const.h	2015-03-27 17:54:06.073139836 +0100
@@ -35,7 +35,7 @@
 #include <errno.h>
 #endif
 
-#include "scst_itf_ver.h"
+#include "iscsi_scst_itf_ver.h"
 
 /*
  * Version numbers, the same as for the kernel.
Index: scst_svn/scripts/generate-kernel-patch
===================================================================
--- scst_svn.orig/scripts/generate-kernel-patch	2015-03-27 17:53:43.381425111 +0100
+++ scst_svn/scripts/generate-kernel-patch	2015-03-27 17:54:06.073139836 +0100
@@ -386,7 +386,7 @@
 	add_file "${f}" "drivers/scst/${f#scst/src/}"
       fi
     done
-    if grep -q /scst_itf_ver.h scst/src/Makefile &&
+    if grep -q /iscsi_scst_itf_ver.h scst/src/Makefile &&
        [ "$s" = "scst_03_public_headers" ]; then
       tmp_itf_ver="$(mktemp /tmp/scst-itf-ver.h.XXXXXXXXXX)"
       cat <<"EOF" >$tmp_itf_ver
@@ -396,7 +396,7 @@
 #define SCST_CONST_INTF_VER "SCST_CONST_INTF_VER"
 #define DEV_USER_INTF_VER "DEV_USER_INTF_VER"
 EOF
-      add_file "$tmp_itf_ver" "include/scst/scst_itf_ver.h"
+      add_file "$tmp_itf_ver" "include/scst/iscsi_scst_itf_ver.h"
       rm -f "$tmp_itf_ver"
     fi
   } |
